truss/patch/calc_patch.py

```diff
@@ -28,8 +35,13 @@ def calc_truss_patch(
         is limited and this usually indicates that the identified change cannot
         be expressed with currently supported patches.
     """
+    if ignore_patterns is None:
+        ignore_patterns = PYCACHE_IGNORE_PATTERNS
```

Comment:
Should we add the pycache patterns to the ignore patterns that are provided in an 'else' here?

Reply:
I'm not sure that we should add it if the ignore pattern is explicitly provided. I can see the value in it, but also I feel it could be surprising. Totally open to change my mind on this.

Reply:
Let's just go with what you have for now and see what we need to change as it's being used